components/climate/coolix.rst

```diff
@@ -36,13 +39,23 @@ Configuration variables:
 - **supports_heat** (*Optional*, boolean): Enables setting cool heat for this climate device. Defaults to ``True``.
 - **sensor** (*Optional*, :ref:`config-id`): The sensor that is used to measure the ambient
   temperature. This is only for reporting the current temperature in the frontend.
+- **receiver_id** (*Optional*, :ref:`config-id`): The remote_receiver id, see: :ref:`coolix-receiver_id`.
 - All other options from :ref:`Climate <config-climate>`.
 
 Advanced options:
 
 - **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
 - **transmitter_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the remote transmitter.
 
+.. _coolix-receiver_id:
+
+Using a receiver
+----------------
+
+When using a receiver it is recommended to put the IR receiver as close as the equipment's IR receiver
+so the chances of receiving (and not receiving) IR signals are the same as of the AC unit.
```

Comment:
I mean yes, that's one way of putting it, but wouldn't it be simpler to say: Just "put the receiver as close as possible"? Is there any reason the chance of receiving and transmitting have to be equal?

Reply:
Ohhh what I'm trying to say is to put the receiver as close as possible to the equipment's original IR receiver, so both receivers next to each other.
This way when a signal is sent by someone using the remote control unit, it really does not depend much where this person is on the room.